lib/dm-rails/setup.rb

```diff
@@ -5,25 +5,33 @@
 
 module Rails
   module DataMapper
+    DATABASE_URL = ENV['DATABASE_URL']
```

Comment:
I wounder if it is the best idea to read the ENV at boot / require time. I think those reads should be done at runtime, from within `Rails::DataMapper.setup`.